wordpress-mobile/WordPress-Android
# Login Rework: new login flow for WPCOM re-auth

**Note: #6164 needs to be merged first.**

This PR enables the new login flow for the WordPress.com re-auth entry point. This entry point is triggered when the app detects that the auth token is no longer valid, at which point it prompts the user to log in again.

Since the prompt is for asking the user to log into WPCOM, the selfhosted flows are not offered. At the epilogue screen, only the "Continue" button is offered.

The final visual touches, the messages and final texts will be done in a future PR.

To test:
* Have the app logged in to WordPress.com and just put it in the background
* Use Calypso on the web and go to https://wordpress.com/me/security/connected-applications. From there, disconnect the Android app from the account.
* Return to the mobile app and relaunch it
* App should ask to login again. Continue and log in using the same WPCOM account.
* Epilogue screen should appear on successful login
